types/nightwatch/test/nightwatch-tests.ts

```diff
@@ -284,6 +284,9 @@ const testPage = {
             .assert.title('Google') // deprecated
             .assert.titleEquals('Google') // new in 2.0
             .assert.visible('@searchBar')
+            .assert.not.titleContains('DuckDuckGo')
+            // @ts-expect-error
+            .assert.not.not.elementPresent('@searchbar')
```

Comment:
This error is going to make all the following chained items into `any`, and thus not actually test anything. You should make this its own chain.